index.html

```diff
@@ -6,170 +6,213 @@
-  <section class="Workssection">
-    <div class="cardBack">
-    <a class="card3" href="#"></a>
-        <h3  class="tonic" >Tonic</h3>
+  <section class="Workssection" id="section-05">
+    <div class="cardBack-05" id ='project-5'>
+         <img class="desktop-05" src="./image/Desktop_03.png" alt="">
+         <img class="Mobile-05" src="./image/Snapshoot Portfolio1 (4).png" alt="">
     <div>
+        <h3  class="tonic3" >Multi-Post Stories</h3>
+        <h3  class="desctonic3" >Uber Navigation</h3>
+        
+         <div>
             <ul class="works">
                 <li class="canopy"> CANOPY</li>
                 <li class="BackendDev">BackendDev</li>
                 <li class="num"> 2015 </li>
             </ul>
-    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
-    </div>
-    <div >
-        <ul class="program1">
-            <li class="Program-skilhtml">html</li>
-            <li class="Program-skilhtml">css</li>
-            <li class="skiljava">javascript</li>
-        </ul>
+            <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+        </div>
+        <div >
+            <ul class="program1">
+                <li class="Program-skilhtml">html</li>
+                <li class="Program-skilhtml">css</li>
+                <li class="skiljava">javascript</li>
+            </ul>
+        </div>
+                <button class="button1 buttonsee"> See Project</button>
     </div>
-        
-    <div class="buttonsee">
-        <button class="button1"> See Project</button>
     </div>
-     </div>
   </section>
   <section class="Workssection">
-    <div class="cardBack">
-    <a class="card4" href="#"></a>
-    
-        <h3  class="tonic3" >Multi-Post Stories </h3>
-        <div>
-    <ul class="works">
-        <li class="canopy"> CANOPY</li>
-        <li class="BackendDev">BackendDev</li>
-        <li class="num"> 2015 </li>
-     </ul>
-    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
-</div>
+    <div class="cardBack2" id="project-4">
 <div >
-    <ul class="program1">
-        <li class="Program-skilhtml">html</li>
-        <li class="Program-skilhtml">css</li>
-        <li class="skiljava">javascript</li>
-    </ul>
-</div>
-   <div class="buttonsee">
-    <button class="button1"> See Project</button>
-</div>
-</div>
-  </section>
-  <section class="Workssection">
-    <div class="cardBack2">
-        
-        <div >
         <h2  class="tonic4" >About Myself </h2>
-    <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+         <p class="section1"> Hello I’m a software developer! I can help you build a product , 
         feature or website Look through some of my work and experience! If you like what you see 
         and have a project you need coded, don’t hestiate to contact me. </p>
```

Comment:
You did an interesting job working on your about section! 💯 However, there are some details that have to be changed in order to complete the project!

- [ ] The first thing to notice here is that the mobile version 📱 is overlapping the contact section and also have extra elements due to some of the changes:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201486137-2bc1d0eb-dadf-49d4-8cd0-5a829d29d62d.png) | ![image](https://user-images.githubusercontent.com/77069011/201486109-237f819c-5b71-433d-84f2-29974725d6dc.png) |

- For the desktop version, please implement the next changes:
  - [ ] Fix the height of the section.
  - [ ] Fix the alignment of the elements.
  - [ ] Add the `"Get My Resume"` button.

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201486251-089f744f-048a-4c36-b53f-be6cedbeb30b.png) | ![image](https://user-images.githubusercontent.com/77069011/201486234-8478d3e9-6b06-4884-ae47-c9aaeb307146.png) |

Remember that the main goal of this project is to follow a given design as closely as possible, so please, stick as much as possible to it! 🤓 🛠️